common-lib/src/main/scala/com/gu/mediaservice/lib/config/MetadataConfig.scala

```diff
@@ -18,10 +24,24 @@ object PhotographersList {
   def list(store: Store) = store.keys.toList.sortWith(_.toLowerCase < _.toLowerCase)
 
   def getPublication(store: Store, name: String): Option[String] = store.get(name)
-}
+
+  def caseInsensitiveLookup(store: Store, lookup: String) =
+    store.map { case (name, pub) =>
+      name.toLowerCase -> (name, pub)
+    }.get(lookup.toLowerCase)
```

Comment:
Good point, this was a repercussion of mucking around with `TreeMap`s.
[Changed](https://github.com/guardian/grid/commit/f4c6525d91899a09625b2737c09ebcb0c0c08032).